src/components/main/StaffList.js

```diff
@@ -44,19 +44,33 @@ class StaffList extends React.Component {
        }
         this.props.onStaffChangeOnUpdate(this.props.staffs)
     }
+   
+    listDelete = (index) =>{
+        this.props.staffs.splice(index, 1)
+        this.props.onStaffChangeOnUpdate(this.props.staffs)
+    }
+   
+    listEdit = () =>
+    {
+        
+    }
 
     render() {
 
         const staffList = this.props.staffs.map((staff, i) =>
             <tr key={staff.id} id={staff.id} >
                 <td>{staff.type}</td>
                 <td>
-                    {staff.quantity}  
-                    <span className="btext" onClick={this.listAdd.bind(staff,i)}>+</span> 
-                    <span className="btext2" onClick={this.listSub.bind(staff,i)}>-</span>
+                    {staff.quantity} 
+                    <button type="button" className="btn btn-outline-primary" onClick={this.listAdd.bind(staff, i)}>&#10133;</button>
+                    <button type="button" className="btn btn-outline-primary" onClick={this.listSub.bind(staff, i)}>&#10134;</button>
```

Comment:
I'm not sure why you did this, because this is not related to your #64 task. This is a task that Emma is going to work on. You can leave it for now, but it'll be changed by Emma, just so you know. Please

Reply:
I fix it, and I'm sorry to bother you!